crates/bevy_render/src/renderer/render_resource_context.rs

```diff
@@ -106,6 +114,33 @@ pub trait RenderResourceContext: Downcast + Send + Sync + 'static {
             .collect();
         PipelineLayout::from_shader_layouts(&mut shader_layouts)
     }
+    /// Reflects the compute pipeline layout from its shaders.
+    ///
+    /// If `bevy_conventions` is true, it will be assumed that the shader follows "bevy shader
+    /// conventions". These allow richer reflection, such as inferred Vertex Buffer names and
+    /// inferred instancing.
+    ///
+    /// If `dynamic_bindings` has values, shader uniforms will be set to "dynamic" if there is a
+    /// matching binding in the list
+    fn reflect_compute_pipeline_layout(
+        &self,
+        shaders: &Assets<Shader>,
+        shader_stages: &ComputeShaderStages,
+        enforce_bevy_conventions: bool,
+    ) -> PipelineLayout {
+        // TODO: maybe move this default implementation to PipelineLayout?
```

Comment:
If we leave the todo here, it would be best to immediately make a follow-up PR or an issue so we can track this progress.